### DISCUSSION TITLE
Import from OneNote?

### DISCUSSION THREAD

**Discussion by StickyPickles** [standardnotes/forum#2704] [General]
If I can import nearly 2Gb over 19 note books from OneNote to Standard Notes, I'm ready to sign up to Extended. However, I'm not finding a method / how-to / tool to do just that. Can someone clarify for me?

If not by a direct route, is there even an indirect route?

Notes include some tables, pdf attachements and inline images.

Any help is much appreciated.

**Comment by JaspalSuri** [comment]
Currently there isn't a way to import notes from the latest version of OneNote into Standard Notes.

If you're using the 2016 version of OneNote, then [this tool](https://github.com/oxhacks/onestandard) might help.
